clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -107,15 +121,19 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
       if (job.tasks) {
         def status = disable ? AbstractEurekaSupport.DiscoveryStatus.OUT_OF_SERVICE : AbstractEurekaSupport.DiscoveryStatus.UP
         task.updateStatus phaseName, "Marking ServerGroup $serverGroupName as $status with Discovery"
-
+        List<String> tasks = job.tasks*.instanceId
```

Comment:
nit ... but I'd perhaps call this `instanceIds` and also replace the other spots we're doing `job.tasks*.instanceId`.

Reply:
thx, updated to take care of this.